tests/assets/test_image_data/small/LUNA-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797/1.3.6.1.4.1.14519.5.2.1.6279.6001.102133688497886810253331438797.mhd

```diff
@@ -1,3 +1,13 @@
-version https://git-lfs.github.com/spec/v1
-oid sha256:9a693678a48ec451b218e1b8d2381e0a5d5ccb6e0f8f1b8100a913eae7bfe34b
-size 392
+ObjectType = Image
```

Comment:
How come this file was wrong / needed a fix?

Reply:
Adds `git-lfs` files to `git`... see #116

Reply:
I see. Thanks! :)